### PR TITLE
build/configs: Fix the flash memory area of the imxrt1020-evk

### PR DESCRIPTION
**Pull request by KimDongEon**
There is not enough memory area to test app. Expanded.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [804e92adaf37309e56259684e4432ae26e82ba0d] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [804e92adaf37309e56259684e4432ae26e82ba0d] - Code Rule Check (C++) OK.

File: build/configs/imxrt1020-evk/scripts/flash-dtcm.ld [hunk 1]
```diff
@@ -57,7 +57,7 @@ MEMORY
 	flash (rx)  : ORIGIN = 0x60000000, LENGTH = 0x00800000
 	sram  (rwx) : ORIGIN = 0x20200000, LENGTH = 0x00020000
 	itcm  (rwx) : ORIGIN = 0x00000000, LENGTH = 0x00010000
-	dtcm  (rwx) : ORIGIN = 0x20000000, LENGTH = 0x00010000
+	dtcm  (rwx) : ORIGIN = 0x20000000, LENGTH = 0x00020000
```
Comment (sunghan-chang) [inline build/configs/imxrt1020-evk/scripts/flash-dtcm.ld]: Total size of internal ram is 256KB. If you want to increase dtcm size, you have to decrease sram size.

**Comment by sunghan-chang** [comment]
Here is no update, let's close. When you need to modify indeed, please reopen.

**Pull request closed without merging**